src/lib/server/use_boost_future.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
```

Comment:
This looks like it comes from somewhere else or is at least based on something. Please reference that source.